support-frontend/app/services/PayPalNvpService.scala

```diff
@@ -59,7 +60,7 @@ class PayPalNvpService(apiConfig: PayPalConfig, wsClient: WSClient) extends Touc
 
   // Takes an NVP response and retrieves a given parameter as a string.
   private def retrieveNVPParam(response: QueryString, paramName: String) =
-    response.paramMap(paramName).headOption match {
+    response.paramMap.getOrElse(paramName, Nil).headOption match {
```

Comment:
did this change to be an Option[List[ ... ]] for some reason?

Reply:
It is a map returning a vector, so we should have always been calling getOrElse. What was happening was that for missing params it was throwing an element not found exception instead of being handled properly by the match statement. This fixes that.